utils/log_writer.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"bufio"
+	"io"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// NewLogWriter pipes stdout/err logs to logrus
+func NewLogWriter(fields log.Fields, level log.Level) io.WriteCloser {
```

Comment:
> ContainerPilot will always allow the application to stream its logs directly to `stdout` and `stderr` since it attaches the streams directly in

Ah, I think I thought you were proposing we use the same mechanism for the application! And sure, because *process* output streams are most likely unstructured I think that's a good approach and not likely to be problematic. Given that, I'm good-to-go with the log levels you had originally. Thanks for getting this thru my noggin. 😀